Sources/Commands/SwiftPackageTool.swift

```diff
@@ -30,44 +30,44 @@ import func POSIX.chdir
 extension PackageToolOptions: XcodeprojOptions {}
 
 private enum Mode: Argument, Equatable, CustomStringConvertible {
-    case Init(InitMode)
+    case initPackage
     case doctor
-    case showDependencies(ShowDependenciesMode)
+    case showDependencies
     case fetch
     case update
     case usage
     case version
-    case generateXcodeproj(String?)
-    case dumpPackage(String?)
+    case generateXcodeproj
+    case dumpPackage
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
         case "init", "initialize":
-            self = try .Init(InitMode(pop()))
+            self = .initPackage
         case "doctor":
             self = .doctor
-        case "show-dependencies", "-D":
-            self = try .showDependencies(ShowDependenciesMode(pop()))
+        case "show-dependencies", "show-deps":
```

Comment:
You're right... I've removed it.  In general we shouldn't have anything that isn't in the help page (which should of course be getting generated mechanically... later).  Do you think we need to keep the -D?